src/codecs.ts

```diff
@@ -871,3 +873,61 @@ export const CookieCsvInput = t.intersection([
 
 /** Type override */
 export type CookieCsvInput = t.TypeOf<typeof CookieCsvInput>;
+
+/// //////////////////////////////////////
```

Comment:
i feel like this may be better in the cli repo? since dont we want to expose these codecs/json schema to customers to implement in their codebase?

Reply:
Is  this not the cli repo? What other repo are you referring to?

Reply:
lol sorry i’m a dummy

Reply:
all good haha